Cql/Cql.Runtime/ValueSets/CqlValueSetFacade.cs

```diff
@@ -46,39 +45,14 @@ public CqlValueSetFacade(string id, IValueSetDictionary valueSets)
         private readonly string _canonical;
         private readonly IValueSetDictionary _valuesets;
 
-        /// <summary>
-        /// Returns <see langword="true"/> if <paramref name="code"/> is in this value set.
-        /// </summary>
-        /// <param name="code">The code to check.</param>
-        /// <returns><see langword="true"/> if <paramref name="code"/> is in this value set.</returns>
-        public bool? IsCodeInValueSet(CqlCode? code)
```

Comment:
Although it makes sense that the return of this function is null when the parameter is null, this actually never happens if you look at the actual invocations.  Also, CqlCode.code being null should not be a reason to return a null here. I think that kind of null  propagation should be in the binding, not here in these basic operators.